mlocal/frags/Makefile.stub

```diff
@@ -53,7 +53,7 @@ e2e-test:
 	$(V)rm -f $(BUILDDIR_ABSPATH)/e2e-cmd-coverage
 	$(V)cd $(SOURCEDIR) && \
 		SINGULARITY_E2E_COVERAGE=$(BUILDDIR_ABSPATH)/e2e-cmd-coverage \
-		scripts/e2e-test -v $(EXTRA_FLAGS)
+		scripts/e2e-test -parallel 16 -v $(EXTRA_FLAGS)
```

Comment:
would it be better to ensure that `GOMAXPROCS` is propagated, and set that to increase the parallelization in CI,  rather than explicitly setting `-parallel 16` here?  I tried running this on a smallish VM on my Mac and it worked but it's pretty much unresponsive during.

Reply:
I hardcoded it for testing in order to spot tests which could conflict and fail with a higher concurrency. Will fix that once everything is OK